pkg/codegen/nodejs/gen.go

```diff
@@ -995,12 +1001,12 @@ func (mod *modContext) sdkImports(nested, utilities bool) []string {
 
 	relRoot := mod.getRelativePath()
 	if nested {
-		imports = append(imports, fmt.Sprintf("import * as inputs from \"%s/types/input\";", relRoot))
-		imports = append(imports, fmt.Sprintf("import * as outputs from \"%s/types/output\";", relRoot))
+		enumsImport := ""
 		containsEnums := mod.pkg.Language["nodejs"].(NodePackageInfo).ContainsEnums
 		if containsEnums {
-			imports = append(imports, fmt.Sprintf("import * as enums from \"%s/types/enums\";", relRoot))
+			enumsImport = ", enums"
 		}
+		imports = append(imports, fmt.Sprintf("import { input as inputs, output as outputs%s } from \"%s/types\";", enumsImport, relRoot))
```

Comment:
nit: If you use a raw string, you don't have to escape the quotes.
```suggestion
		imports = append(imports, fmt.Sprintf(`import { input as inputs, output as outputs%s } from "%s/types";`, enumsImport, relRoot))
```